### ISSUE TITLE
4k Format not showing 

### ISSUE THREAD

**Issue by baadshah0786** [yuliskov/SmartTube#3656]
![IMG20240710193722](https://github.com/yuliskov/SmartTube/assets/87961897/437e5a0b-c0f1-4c54-af3f-e661ae91c611)
I m playing 4k videos but max resulation 1080p shown only plz fix it up bro

**Comment by baadshah0786** [comment]
It's chance to fix or not

**Comment by TecnikOfficial** [comment]
yes the dev yuliskov is tryin he already released beta version for 4k fix but i think you should wait for stable version

**Comment by dylangerdaly** [comment]
Are they gonna fix this? I remember hearing Google were going to gate anything over1080p to Premium

**Comment by roastduckkiller** [comment]
Same problem here.

**Comment by josejoa** [comment]
> Are they gonna fix this? I remember hearing Google were going to gate anything over1080p to Premium

You can see many videos 4K for free right now

**Comment by sentix001** [comment]
Same here both smartube beta and and smartube nor working 4k

Browser with an adblocker works fine
